public/index.html

```diff
@@ -56,6 +56,7 @@
                         <label for="phone_number">Phone number</label>
                         <input class="form-control input-lg" type="phone" name="phone_number" id="phone_number">
                     </div>
+                    <input type="hidden" name="campaign" id="campaign" value="some_campaign">
```

Comment:
Was talking to deray about how they wanna do bucketing. One possibility we discussed was hosting this form at special event- or campaign-specific URLs. e.g. `staywoke.org/in/inauguration` and then we'd consume that last param as the campaign name. whatever form it takes, i think the client should be able to set the campaign. (similarly, the SMS app could accept the campaign as the initial contact keyword, e.g. "text 'inauguration' to 123-456-7890").

so maybe we address this concern by having a whitelist of valid campaigns on the backend. but maybe we should accept any invalid campaign, prefix it with 'unknown:', and then do some data cleanup to address common typos and such.